README.md

```diff
@@ -35,32 +35,52 @@ Understanding how humans leverage semantic knowledge to navigate unfamiliar envi
 
 ## :hammer_and_wrench: Installation
 
-### Getting Started
+### Getting Started: Environment
 
 Create the conda environment:
 ```bash
 conda_env_name=vlfm
 conda create -n $conda_env_name python=3.9 -y &&
 conda activate $conda_env_name
 ```
+
+Check compiled cuda version/available cuda version and install the closest available corresponding torch. *This will be important for successful GroundingDINO installation*
+
 If you are using habitat and are doing simulation experiments, install this repo into your env with the following:
 ```bash
 pip install -e .[habitat]
 ```
+
 If you are using the Spot robot, install this repo into your env with the following:
 ```bash
 pip install -e .[reality]
 ```
-Install all the dependencies:
+
+Clone the below dependencies into the root directory of this repo:
 ```bash
-git clone git@github.com:IDEA-Research/GroundingDINO.git
-git clone git@github.com:WongKinYiu/yolov7.git  # if using YOLOv7
+git clone https://github.com/IDEA-Research/GroundingDINO.git
+git clone https://github.com/WongKinYiu/yolov7.git  # if using YOLOv7
+git clone https://github.com/facebookresearch/habitat-lab
```

Comment:
This is already under `pyproject.toml`, installable under `pip install -e .[habitat]`

Reply:
Unfortunately for some reason in my Windows and Linux environment even though `pip install -e .[habitat]` ran without any errors it didn't git clone the repo's into the directory and files from the yolov7 and groundingdino were required to run the models from what I rememeber